Spec/Auth.swift

```diff
@@ -1106,6 +1106,36 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
+                tokenParams.clientId = "newClientId"
+                tokenParams.ttl = 2000
+                tokenParams.capability = "{ \"test:*\":[\"test\"] }"
+                
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.createTokenRequest(tokenParams, options: authOptions) { tokenRequest, error in
```

Comment:
Yes @mattheworiordan, we're forced to do this every time we have more than one argument. It's just how Objective-C works. (When the last argument is a callback, as a syntactic convenience Swift lets you omit its name and pass it kind of like a Ruby block, that's why you don't see its name.)